io.github.hakuneko.HakuNeko.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.hakuneko.HakuNeko</id>
+	<launchable>io.github.hakuneko.HakuNeko.desktop</launchable>>
```

Comment:
This should ideally go upstream.

Reply:
Could I put the metainfo file anywhere in the Github Repo or is there a specific place I have to put the file in the upstream Github Repo?

Reply:
It could be put anywhere really. Just install it during the build or as a post-install here until electron-builder adds support of it.

Reply:
I think I could maintain it here post-install until electron-builder adds metadata support.